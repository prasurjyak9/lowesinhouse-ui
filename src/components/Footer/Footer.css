.footer-alt

  {

      padding-bottom: 10px;

      padding-top: 10px;

      background-color:#e2e2e2;

      color: #1e1e1e;

  }

  .text-white {

    color: #fff;

  }

  .text-center {

    text-align: center;

  } 

  .mt-4 {

    margin: 0;

    margin-top: 1.5rem;

    color: white;

  }

  .bg-footer {

    color: #1e1e1e;

    position: fixed;

    width: 100%;

    bottom: 0;

  }

  .footer-alt {

    color:#1e1e1e;

  }

  

  @media (min-width: 768px) {

    .container {

      max-width: 720px;

    }

  }

  

  @media (min-width: 576px) {

    .container {

      max-width: 540px;

      width: 100%;

      padding-right: var(--bs-gutter-x, 0.75rem);

      padding-left: var(--bs-gutter-x, 0.75rem);

      margin-right: auto;

      margin-left: auto;

    }
    .bg-footer {

        background-color: #012169;
  
        width: 100%;
  
        bottom: 0;
  
      }
  
      
  
  
  
    }
  
    
  
    @media (min-width: 992px) {
  
      .container {
  
        max-width: 960px;
  
      }
  
    }   
  
    @media (min-width: 768px) {
  
        .container {
  
          max-width: 720px;
  
        }
  
      }
  
       
  
      @media (min-width: 576px) {
  
        .container {
  
          max-width: 540px;
  
          width: 100%;
  
          padding-right: var(--bs-gutter-x, 0.75rem);
  
          padding-left: var(--bs-gutter-x, 0.75rem);
  
          margin-right: auto;
  
          margin-left: auto;
  
        }
  
        
  
      }
  
      
  
      @media (min-width: 992px) {
  
        .container {
  
          max-width: 960px;
  
        }
  
      }