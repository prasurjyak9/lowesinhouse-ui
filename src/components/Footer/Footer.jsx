import React from "react";

import './Footer.css';



const Footer = () => {

    return (

        <div className="footer-wrap" data-testid="footer-test">

            <footer className="section bg-footer" data-testid="footer" >

                <div className="text-center mt-5">

                    <p className="footer-alt">2022 Â© Lowe's Pvt Limited, All Rights Reserved</p>

                </div>

            </footer>

        </div>

    )

}

export default Footer;