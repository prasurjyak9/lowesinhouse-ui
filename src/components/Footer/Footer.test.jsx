import React from 'react'

import { render, fireEvent, screen } from '@testing-library/react'

import '@testing-library/jest-dom/extend-expect'

import { MemoryRouter } from 'react-router-dom';

import Footer from './Footer';



describe('Input value', () => {

    

    it("test footer ", () =>{

        render(<MemoryRouter> <Footer/> </MemoryRouter>)

        expect(screen.getByTestId("footer-test")).toMatchSnapshot()

    })

})