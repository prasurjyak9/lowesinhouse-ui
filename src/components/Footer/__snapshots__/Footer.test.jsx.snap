// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input value test footer  1`] = `
<div
  class="footer-wrap"
  data-testid="footer-test"
>
  <footer
    class="section bg-footer"
    data-testid="footer"
  >
    <div
      class="text-center mt-5"
    >
      <p
        class="footer-alt"
      >
        2022 Â© Lowe's Pvt Limited, All Rights Reserved
      </p>
    </div>
  </footer>
</div>
`;
